iree/base/internal/atomics_test.cc

```diff
@@ -14,6 +14,9 @@
 
 #include "iree/base/internal/atomics.h"
 
+#include <cstddef>
+#include <cstdint>
```

Comment:
Is there a rule of thumb to use for `cstdint` vs `stdint.h`? `iree/base/internal/atomic_slist.h` uses `stdint.h` while this test file uses `cstdint`.
https://stackoverflow.com/questions/13642827/cstdint-vs-stdint-h

Reply:
c uses c includes (<stdfoo.h>), c++ uses c++ includes (<cstdfoo>) - atomic_slist.h is c, test.cc files are c++
we could just use the c ones everywhere, but then clang-tidy complains